vpr/src/route/route_common.cpp

```diff
@@ -568,7 +568,7 @@ static t_trace_branch traceback_branch(int node, int target_net_pin_index, std::
         t_trace* prev_ptr = alloc_trace_data();
         prev_ptr->index = inode;
         prev_ptr->net_pin_index = OPEN; //Net pin index is invalid for Non-SINK nodes
-        prev_ptr->iswitch = device_ctx.rr_nodes.edge_switch(iedge);
+        prev_ptr->iswitch = rr_graph.edge_switch(iedge);
```

Comment:
Does ``rr_graph.nodes().edge_switch(iedge)`` work here? If we can reuse, it is better to reuse.

Reply:
@tangxifan 
I am facing some unknown errors for failing of basics tests. I will see it now.